var/spack/repos/builtin/packages/turbine/package.py

```diff
@@ -40,6 +40,7 @@ class Turbine(AutotoolsPackage):
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
     depends_on('libtool', type='build')
+    depends_on('m4', type=('build', 'run'))
```

Comment:
@jmjwozniak generally `m4` is only a build dependency, do you expect that this package will need it after building?

Reply:
m4 is a build and run dependency.  We use m4 at run time to generate scheduler job files for the workflows.